docs/server/commands.rst

```diff
@@ -893,6 +893,282 @@ and the process will start a watchdog to track any client-side scripts for
 changes. Use this only when developing Pootle.
 
 
+.. _commands#pootle-fs:
+
+Pootle_fs
+---------
+
+
+
+
+
+Path options
+------------
+
+.. django-admin-option:: --pootle_path -P
```

Comment:
I fixed this previously didn't I?  Short then long, just like every single man page and --help page